templates/_helpers.tpl

```diff
@@ -46,54 +46,57 @@ app.kubernetes.io/managed-by: {{ .Release.Service }}
 {{/*
 Selector labels
 */}}
-{{- define "fcrepo.selectorLabels" -}}
+{{- define "fcrepo.selectorLabels" }}
 app.kubernetes.io/name: {{ include "fcrepo.name" . }}
 app.kubernetes.io/instance: {{ .Release.Name }}
 {{- end }}
 
 {{/*
 Create the name of the service account to use
 */}}
-{{- define "fcrepo.serviceAccountName" -}}
+{{- define "fcrepo.serviceAccountName" }}
 {{- if .Values.serviceAccount.create }}
 {{- default (include "fcrepo.fullname" .) .Values.serviceAccount.name }}
 {{- else }}
 {{- default "default" .Values.serviceAccount.name }}
 {{- end }}
 {{- end }}
 
-{{- define "fcrepo.postgresql.fullname" -}}
-{{- $name := default "postgresql" .Values.postgresql.nameOverride -}}
-{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" -}}
-{{- end -}}
+{{- define "fcrepo.postgresql.fullname" }}
+{{- $name := default "postgresql" .Values.postgresql.nameOverride }}
+{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" }}
+{{- end }}
 
 {{/*
 Return PostgreSQL host
 */}}
-{{- define "fcrepo.postgresql.host" -}}
-{{- include "fcrepo.postgresql.fullname" . }}
-{{- end -}}
+{{- define "fcrepo.postgresql.host" }}
+{{- if .Values.postgresql.postgresqlHost }}
+    {{- .Values.postgresql.postgresqlHost }}
+{- else }}
+    {{- include "fcrepo.postgresql.fullname" . }}
+{{- end }}
 
 {{/*
 Return PostgreSQL username
 */}}
-{{- define "fcrepo.postgresql.username" -}}
+{{- define "fcrepo.postgresql.username" }}
 {{- if .Values.global.postgresql.postgresqlUsername }}
-    {{- .Values.global.postgresql.postgresqlUsername -}}
-{{- else -}}
-    {{- .Values.postgresql.postgresqlUsername -}}
-{{- end -}}
-{{- end -}}
+     {{- .Values.global.postgresql.postgresqlUsername }}
+{{- else }}
+     {{- .Values.postgresql.postgresqlUsername }}
+{{- end }}
+{{- end }}
 
 {{/*
 Return PostgreSQL password
 */}}
-{{- define "fcrepo.postgresql.password" -}}
+{{- define "fcrepo.postgresql.password" }}
 {{- if .Values.global.postgresql.postgresqlPassword }}
     {{- .Values.global.postgresql.postgresqlPassword }}
-{{- else if .Values.postgresql.postgresqlPassword -}}
+{{- else if .Values.postgresql.postgresqlPassword }}
     {{- .Values.postgresql.postgresqlPassword }}
-{{- else -}}
-    {{- randAlphaNum 10 -}}
-{{- end -}}
-{{- end -}}
+{{- else }}
+    {{- randAlphaNum 10 }}
```

Comment:
i wondered about this earlier. does this work, or just create a new random password that's different from the random one generated by the postgres chart?

it can be out-of-scope here, but as long as we are chatting. :)

Reply:
I agree, I don't think this does anything useful.